lldb/source/Plugins/Language/Swift/SwiftUnsafeTypes.cpp

```diff
@@ -170,13 +252,18 @@ lldb_private::formatters::swift::UnsafeBufferPointerSyntheticFrontEnd::
   m_ptr_size = process_sp->GetAddressByteSize();
   m_order = process_sp->GetByteOrder();
 
+  llvm::StringRef valobj_type_name(valobj_sp->GetTypeName().GetCString());
+  bool is_raw = valobj_type_name.contains("Raw");
```

Comment:
It would be better to have a factory method that makes the right version as a UP.  Then you could set it into your ivar here and in the Summary you could also use that to avoid the ugly if (Raw) {make_raw; do stuff; } else {make_not_raw; do same stuff } construct.

Reply:
Will do!